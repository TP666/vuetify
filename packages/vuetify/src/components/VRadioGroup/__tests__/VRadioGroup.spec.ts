// Components
import VRadioGroup from '../VRadioGroup'

// Utilities
import {
  mount,
  Wrapper
} from '@vue/test-utils'

describe('VRadioGroup.ts', () => {
  type Instance = InstanceType<typeof VRadioGroup>
  let mountFunction: (options?: object) => Wrapper<Instance>

  beforeEach(() => {
    mountFunction = (options = {}) => {
      return mount(VRadioGroup, {
        ...options
      })
    }
  })

  it('should match snapshot', async () => {
    const wrapper = mountFunction()

    expect(wrapper.html()).toMatchSnapshot()
  })
})
