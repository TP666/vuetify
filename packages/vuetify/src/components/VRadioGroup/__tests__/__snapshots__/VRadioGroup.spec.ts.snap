// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VRadioGroup.ts should match snapshot 1`] = `

<div class="v-item-group v-input--selection-controls v-input--radio-group v-input--radio-group--column">
</div>

`;
